compiler/ballerina-lang/src/main/java/org/ballerinalang/util/diagnostic/DiagnosticErrorCode.java

```diff
@@ -178,6 +178,8 @@ public enum DiagnosticErrorCode implements DiagnosticCode {
     INVALID_SPREAD_OP_FIXED_MEMBER_EXPECTED("BCE2134", "invalid.spread.operator.fixed.member.expected"),
     CANNOT_INFER_TYPE_FROM_SPREAD_OP("BCE2135", "cannot.infer.type.from.spread.op"),
     TUPLE_AND_EXPRESSION_SIZE_DOES_NOT_MATCH("BCE2136", "tuple.and.expression.size.does.not.match"),
+    CANNOT_SPECIFY_NAMED_ARG_FOR_FIELD_OF_INCLUDED_RECORD_PARAM("BCE2137",
+            "cannot.specify.named.argument.for.field.of.included.record.parameter"),
```

Comment:
This isn't comprehensive and kinda misleading, right? Shall we update this to reflect the actual error?

Reply:
Changed